sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -0,0 +1,228 @@
+import os
+import sys
+import subprocess
+
+import pytest
+import unittest
+from imp import load_source
+
+if sys.version_info >= (3, 3):
+    from unittest.mock import MagicMock
+else:
+    from mock import MagicMock, patch
+
+from sonic_py_common import daemon_base
+from swsscommon import swsscommon
+from .mock_swsscommon import Table
+
+
+daemon_base.db_connect = MagicMock()
+swsscommon.Table = MagicMock()
+swsscommon.ProducerStateTable = MagicMock()
+sys.modules['sonic_y_cable'] = MagicMock()
+sys.modules['sonic_y_cable.y_cable'] = MagicMock()
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "xcvrd")
+helper_file_path = os.path.join(scripts_path, "xcvrd_utilities"+"/y_cable_helper.py")
+sys.path.insert(0, modules_path)
+
+"""
+if __name__ == '__main__' and __package__ is None:
+    from os import sys, path
+    sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+"""
+os.environ["XCVRD_UNIT_TESTING"] = "1"
+load_source('y_cable_helper', scripts_path + '/xcvrd_utilities/y_cable_helper.py')
+from y_cable_helper import *
+from xcvrd.xcvrd import *
+
+PY3x = sys.version_info >= (3, 0)
+PYvX_DIR = "py3" if PY3x else "py2"
+
+PYTHON_INTERPRETTER = "python3" if PY3x else "python2"
```

Comment:
Typo: `PYTHON_INTERPRETER`. Also, is this used?

Reply:
fixed, removed